ct/spoolman.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/tteck/Proxmox/main/misc/build.func)
+# Copyright (c) 2021-2024 tteck
+# Author: MickLesk (Canbiz)
```

Comment:
Co-Author

Reply:
my mistake, I forgot to update my template :-)